### DISCUSSION TITLE
How to configure odoo to connect to a postgresql database hosted on digitalocean?

### DISCUSSION THREAD

**Discussion by vasi26ro** [Tecnativa/doodba-copier-template#384] [Q&A]
I want to test the performance of odoo with the database hosted on digitalocean.
How to configure the odoo image to use a database that is outside of the docker environment?

Thank you

**Comment by vasi26ro** [comment]
I found a way:

Add: `networks: *public` to `odoo` service in `docker-compose.yml`
Add the following lines to `odoo/custom/conf.d/odoo.conf`:

```
[options]
db_user = 
db_password = 
db_host = 
db_port = 
db_name =
```

**Comment by pedrobaeza** [comment]
In prod.yaml, in the environment section of `odoo`, put
```
PGHOST: &dbhost
     <your_digitalocean_host>
```

and if you want backup, in the environment section of such container:

```
PGHOST: *dbhost
```

Finally, remove the `db` service from the file.
